electron-notify/electron-notify.d.ts

```diff
@@ -0,0 +1,68 @@
+// Type definitions for electron-notify v0.1.0
+// Project: https://github.com/hankbao/electron-notify
+// Definitions by: Daniel Pereira <https://github.com/djpereira>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference path="../github-electron/github-electron.d.ts" />
+
+declare namespace ElectronNotify {
+
+  interface ICloseNotificationEvent {
```

Comment:
Can you stick the interfaces in the 'electron-notify' module, and export them so they can be used?